src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2815,6 +2812,9 @@ private int[] SerializeMethodBodies(BlobBuilder ilBuilder, PdbWriter pdbWriterOp
 
             var encoder = new MethodBodiesEncoder(ilBuilder);
 
+            var mvidStringHandle = default(UserStringHandle);
```

Comment:
It doesn't look like this variable is used in any meaningful way. It also looks like none of the consumers of SerializeMethodBody take advantage of the parameter either. Consider removing the local and the parameter.